src/applications/facility-locator/tests/00-breadcrumbs.e2e.spec.js

```diff
@@ -0,0 +1,38 @@
+const E2eHelpers = require('../../../platform/testing/e2e/helpers');
+const Timeouts = require('../../../platform/testing/e2e/timeouts.js');
+const FacilityHelpers = require('./facility-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    FacilityHelpers.initApplicationMock();
+
+    client
+      .url(`${E2eHelpers.baseUrl}/facilities/`);
+
+    E2eHelpers.overrideSmoothScrolling(client);
+    FacilityHelpers.initApplicationMock();
+
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible('.facility-locator', Timeouts.slow);
+
+    client
+      .waitForElementVisible('.va-nav-breadcrumbs', Timeouts.normal)
+      .waitForElementVisible('.va-nav-breadcrumbs-list', Timeouts.normal)
+      .waitForElementVisible('a[aria-current="page"', Timeouts.normal);
+
+    client.expect.element('.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]').to.be.present;
+    client.expect.element('.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]').to.have.css('pointer-events').which.equal('none');
+
+    // Mobile test, most common size
+    client.resizeWindow(375, 6667);
+
+    client.waitForElementVisible('.va-nav-breadcrumbs-list', Timeouts.normal);
+    client.expect.element('.va-nav-breadcrumbs-list li:not(:nth-last-child(2))').to.have.css('display').which.equal('none');
+    client.expect.element('.va-nav-breadcrumbs-list li:nth-last-child(2)').to.have.css('display').which.equal('inline-block');
+
+    // Reset default size
+    client.resizeWindow(1024, 768);
+
+    client.end();
+  });
```

Comment:
These E2E tests are great, but I'm not sure how necessary it is to test each application's implementation since they share a common component now.

Reply:
The form [e2e tests](https://github.com/department-of-veterans-affairs/vets-website/blob/18157605a659fae278754771efba6ab050eb60b8/src/applications/pensions/tests/00-all-fields.e2e.spec.js#L26) include checks to ensure that the progress bar presents the appropriate information, it seems consistent to use e2e tests to ensure that the breadcrumbs display the appropriate information as well.

Reply:
@cehsu Great call! I updated the e2e tests to check not only for presence of the breadcrumbs, but the text of the terminal element on standard and mobile breadcrumbs. Makes the tests more comprehensive.